Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# Vehicle to emplacement

## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

Attempt to allow AI to man gunner & commander seats in vehicles that were left near friendly markers

You can block ai from accessing certain vehicles, but same deal with statics, when you reload a save, AI blockage will reset. 

I don't allow ai to drive vehicles(yet(or ever)) because I didn't figure out how to store their original position

Also no air vehicles, because... well no driver = no point and idk, don't think it's a good thing to do, maybe someday and in different form

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #254!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [ ] No
2. [x] Yes (I need help with this one)

### How can the changes be tested?
Steps:
Spawn vehicle, leave it near friendly marker. Save and reload, it should save a vehicle
********************************************************
Notes:
